prime-dotnet-webapi/Services/EnrolleePaperSubmissionService.cs

```diff
@@ -264,5 +264,130 @@ public async Task FinalizeSubmissionAsync(int enrolleeId)
             _context.Set<T>().RemoveRange(oldItems);
             _context.Set<T>().AddRange(itemList);
         }
+
+        public async Task<bool> IsPotentialPaperEnrolleeReturneeAsync(DateTime dateOfBirth)
+        {
+            var confirmedLinks = await _context.EnrolleeLinkedEnrolment
+                .AsNoTracking()
+                .Where(cl => cl.Confirmed)
+                .Select(cl => cl.PaperEnrolleeId)
+                .ToListAsync();
+
+            return await _context.Enrollees
+                .AsNoTracking()
+                .AnyAsync(
+                    e => e.GPID.StartsWith(PaperGpidPrefix)
+                    && !confirmedLinks.Contains(e.Id)
+                    && e.DateOfBirth.Date == dateOfBirth.Date
+                );
+        }
+
+        public async Task<IEnumerable<Enrollee>> GetPotentialPaperEnrolleeReturneesAsync(DateTime dateOfBirth)
+        {
+            return await _context.Enrollees
+                .AsNoTracking()
+                .Where(
+                    e => e.GPID.StartsWith(PaperGpidPrefix)
+                    && e.DateOfBirth.Date == dateOfBirth.Date
+                    && !_context.EnrolleeLinkedEnrolment.Any(
+                        ele => ele.Confirmed
+                        )
```

Comment:
You updated the logic in `IsPotentialPaperEnrolleeReturneeAsync` but not here

Reply:
That one was supposed to be fixed alongside the other one but looks like it wasn't. Fixed now for sure